### ISSUE TITLE
Feature Idea: Hide Ratings

### ISSUE THREAD

**Issue by grantoesterling** [jgchk/better-rym#152]
I think it'd be cool to have an optional feature that hides album / track ratings til after you have rated something. I often find myself being influenced by the track ratings or my friend's ratings to rate something higher/lower or pay more attention to specific tracks

If I eventually have time, I might add this myself. Though I'm not sure if this thing can support toggle-able features...

**Comment by jgchk** [comment]
Good idea. This is something I've needed but never been able to put into words.

We have toggleable features, just no dedicated centralized interface for them. We'd either need to build out that interface (probably a dropdown when you click the extension icon) or integrate the setting into the release page itself.
